.github/workflows/browserstack-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: BrowserStack
+
+# BrowserStack tests are run against the deployed GitHub pages URL
+# The closest event we can use to trigger when these are updated is page_build.
+# on: [page_build]
+on: [push]
+
+jobs:
+  desktop-browsers:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - config: 'Windows_10_latest'
+            browser: 'chrome'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'Windows_10_latest'
+            browser: 'edge'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'Windows_7_latest'
+            browser: 'firefox'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'OSX_Mojave_latest'
+            browser: 'safari'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'iPhone11_13'
```

Comment:
Would be nice to put the two mobile tests into a separate job called 'mobile-browsers'

Reply:
I did start with it this way and it’s quite a lot more boilerplate, but let me push a change and we can compare the two.